pages/linux/enum4linux.md

```diff
@@ -0,0 +1,24 @@
+# enum4linux
+
+> Enum4linux is a tool for enumerating information from Windows and Samba systems.
+> It attempts to offer similar functionality to enum.exe formerly available from www.bindview.com.
+
+- Try to enumerate using all methods:
+
+`enum4linux -a remote_host`
+
+- Enumerate using login:
+
+`enum4liux -u user_name -p password remote_host`
+
+- Enumerate host for usernames:
```

Comment:
List usernames from a given host.

Reply:
@agnivade I edited your comment to make it a suggestion. I hope that's okay. :p